src/Symfony/Component/PropertyAccess/PropertyPath.php

```diff
@@ -100,6 +109,12 @@ public function __construct($propertyPath)
                 $this->isIndex[] = true;
             }
 
+            if (substr($element, 0, 1) === '?') {
```

Comment:
I think it should be the last char instead, to match the [php RFC](https://wiki.php.net/rfc/nullsafe_calls) and feature request 😉

Reply:
You are right I will change it,